### PR TITLE
layer: add use of "entries" for "files" in tar

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

**Comment by vbatts** [comment]
cc @stevvooe

**Comment by vbatts** [comment]
well, my DCO doesn't suffice for carrying your commit ...

**Comment by philips** [comment]
@vbatts rewrite the commit author and force push, it is a trivial addition

**Comment by wking** [comment]
On Thu, Sep 08, 2016 at 02:54:13PM -0700, Brandon Philips wrote:
> @vbatts rewrite the commit author and force push, it is a trivial addition

Or add a Signed-off-by for @stevvooe above yours [1], assuming he
gives permission to do so.

[1]: https://git.kernel.org/cgit/git/git.git/tree/Documentation/SubmittingPatches?h=v2.10.0#n289

File: layer.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Removals are represented using "[whiteout](#whiteouts)" file entries (See [Repre
 
 ### File Types
 
-Throughout this document section, the use of word "files" includes:
+Throughout this document section, the use of word "files" or "entries" includes:
```
Comment (wking) [inline layer.md]: “of word” is awkward.  Maybe:

> Throughout this document section, the words “files” and “entries” include:

I'm also not sure about “document section”.  I expect we want something scoped to the whole layer.md, not just the “File Types” h3.  How about:

> Throughout the [layer documentation](#image-layer-filesystem-changeset), the words “files” and “entries” include:

**Comment by stevvooe** [comment]
@philips @vbatts Ping me and I'll carry it!

**Comment by wking** [comment]
On Thu, Sep 08, 2016 at 03:44:25PM -0700, Stephen Day wrote:
> @philips @vbatts Ping me and I'll carry it!

@vbatts should be able to use [1] to give all maintainers push access
to this particular branch.

[1]: https://github.com/blog/2247-improving-collaboration-with-forks

**Comment by vbatts** [comment]
the "Allow edits from maintainers" is already checked.

updated the Author.

**Comment by philips** [comment]
LGTM

**Comment by stevvooe** [comment]
LGTM

File: layer.md (hunk 1 above)
Comment (jonboulle) [inline layer.md]: I am going to follow this up

**Comment by jonboulle** [comment]
lgtm

**Pull request merged**